ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarMultiThreadingManager.java

```diff
@@ -326,11 +337,19 @@ public void join()
 
    public void stop()
    {
+      estimatorROS2Node.stopSpinning();
+      controllerROS2Node.stopSpinning();
       hardwareCommunicationInterface.stop();
    }
 
    public void destroy()
    {
+      estimatorROS2Node.destroy();
```

Comment:
to match the old functionality, this needs to call `this.stop()` in the first line of the method.

Reply:
I looked and it's effectively useless.